src/components/PostPageDetail/index.js

```diff
@@ -678,7 +677,7 @@ const PostPageDetailIdComponent = (props) => {
   );
 };
 
-export default withInteractionsManaged(React.memo(PostPageDetailIdComponent));
+export default PostPageDetailIdComponent;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Removing `withInteractionsManaged` and `React.memo` could potentially impact performance if this component is frequently re-rendered. If there's no specific reason for this change, consider reverting it.
```diff
- export default withInteractionsManaged(React.memo(PostPageDetailIdComponent));
+ export default PostPageDetailIdComponent;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->